src/twisted/web/client.py

```diff
@@ -753,6 +779,9 @@ def getPage(url, contextFactory=None, *args, **kwargs):
         *args, **kwargs).deferred
 
 
+
+@deprecated(Version("Twisted", 16, 7, 0),
+            "https://pypi.org/project/treq/ or twisted.web.client.Agent")
```

Comment:
(optional) Since the deprecation message is being used multiple times in this file, what would you think of pulling  into a private module variable?